docs/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,32 @@
+.md-search__form {
+    border-radius: 24px;
+}
+
+.md-footer-meta {
+    display: none;
+}
+
+.demo-button {
+    align-items: center;
+    background-color: #CB6E00;
+    border-radius: 24px;
+    color: #FFFFFF;
+    cursor: pointer;
+    font-family: Inter, sans-serif;
```

Comment:
How are we importing the font?

Reply:
We are not importing fonts for now.  Can be taken up as an addon with UI updation like suggested